theano/compile/tests/test_pfunc.py

```diff
@@ -810,15 +810,15 @@ def test_partial_input_aliasing_affecting_inplace_operations(self):
         #   c does not share memory with a
 
         f = theano.function(
-                [theano.In(x,  mutable=True),
-                 theano.In(y,  mutable=True),
-                 theano.In(z,  mutable=True),
-                 theano.In(m1, mutable=True),
-                 theano.In(m2, mutable=True),
-                 theano.In(m3, mutable=True)],
-                (theano.dot((x * 2), m1)
-                    + theano.dot((y * 3), m2)
-                    + theano.dot((z * 4), m3)))
+            [theano.In(x, mutable=True),
+                theano.In(y, mutable=True),
+                theano.In(z, mutable=True),
+                theano.In(m1, mutable=True),
+                theano.In(m2, mutable=True),
+                theano.In(m3, mutable=True)],
+            (theano.dot((x * 2), m1) +
+                theano.dot((y * 3), m2) +
+                theano.dot((z * 4), m3)))
```

Comment:
The lines inside the list should not be further indented and you can remove the parenthesis in the second block and avoid one line break:
```python
[theano.In(x, mutable=True),
 theano.In(y, mutable=True),
 theano.In(z, mutable=True),
 theano.In(m1, mutable=True),
 theano.In(m2, mutable=True),
 theano.In(m3, mutable=True)],
(theano.dot((x * 2), m1) + theano.dot((y * 3), m2) +
 theano.dot((z * 4), m3)))
```